test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -372,7 +372,12 @@
         (testing "timestamp with time zone columns"
           (mt/with-report-timezone-id "UTC"
             (testing "convert to +09:00"
-              (is (= ["2004-03-19T02:19:09Z" "2004-03-19T11:19:09+09:00"]
+              (is (= (case driver/*driver*
+                       ;; TIMEZONE FIXME
+                       ;; for some reasons redshift is inserting the dt_tz column with timezone = UTC, it should be
+                       ;; Asia/Ho_Chi_Minh.
+                       :redshift ["2004-03-19T09:19:09Z" "2004-03-19T18:19:09+09:00"]
```

Comment:
Does this need to be fixed?

Reply:
I think it's separate from the purpose of this PR.

It is a problem with how we created test data for Redshift.

For postgres, when we insert `"2004-03-19 09:19:09+07:00[Asia/Ho_Chi_Minh]"` with column type = `timetamp with time zone`, since Postgres will convert it to UTC for storage, it'll become `2004-03-19 02:19:09Z`

But for some reasons it's not the same as redshift, the same value inserted to redshift was ignored in the timezone, so we're storing `2004-03-19 09:19:09Z` for redshift.

Reply:
I'll try to debug it, but I don't think it's a blocking issue for this PR.

Reply:
it's fixed ! https://github.com/metabase/metabase/pull/26633/commits/6d0772cfdbf97d16ac54103f705039e8772f86aa